### DISCUSSION TITLE
Visualizing solvation shells in MolecularNodes

### DISCUSSION THREAD

**Discussion by janash** [janash/MolecularNodes#3] [Ideas]
This is a discussion thread for how to visualize solvation shells in MolecularNodes.

Option 1: The user inputs the solvent and shell information in when the trajectory is loaded.
============================================================
Information at load time: MD topology + trajectory. Solvent, solute selection; shell selection; frame number.
Advantages: This is the easiest route to go. 
Disadvantages: This route lacks flexibility and will require reloading the trajectory into a new blender object every time a new solvation shell or frame needs to be visualized.

Option 2: The user loads an MD trajectory with solvent information and can dynamically select the shell and frame.
==========================================================================
Information at load time: MD topology + trajectory. Solvent, solute selection.
Advantages: The most flexible and probably the solution many would expect.
Disadvantages: Unclear if possible. Might require altering a lot of the MolecularNodes package .

@kimjoyc - anything I'm missing? Is this clear/correct?

We could get Option 1 working as a minimum viable product (MVP) and move on to trying to implement Option 2.

**Comment by kimjoyc** [comment]
Can I incorporate this as a feedback from you for Assignment 3?
